resources/files/sysv-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+unset LD_LIBRARY_PATH
+unset LD_PRELOAD
```

Comment:
It doesn't look we have a case for `darwin` in the acceptance test - can you add one or was that prohibitively costly? Seems like that platform seems to edge-case us all the time..